LokiLabs/microsoft-graph-explorer-v4
# AB#39419 Interns/jerryxihe/39419 app mode hide token tab

ADO Board Link: [Task #39419](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_workitems/edit/39419)

## Overview

* Remove "Access token" tab and rename "Modify permission (preview)" to "Required permissions" from view when in application mode.
* In "Required permissions" page - string should include "To grant these permissions, please install the Graph Explorer sample Teams app against the desired resource."

### Demo

![image](https://user-images.githubusercontent.com/46834951/128227820-3cacc99d-e927-4373-a2a4-4631d819952b.png)

### Notes

## Testing Instructions

* Check out this branch.
* Run `nvm use --lts`
* Run `npm install && npm start`
* Log in and switch to app mode
* Check that all the changes described in `Overview` show up
